tavern_server/src/db_tables.sql

```diff
@@ -0,0 +1,676 @@
+--enum types
+CREATE TYPE Gender AS ENUM (
+	'Male', 
+	'Female', 
+	'Other'
+);
+
+CREATE TYPE Size AS ENUM (
+	'Fine', 
+	'Diminutive', 
+	'Tiny', 
+	'Small', 
+	'Medium', 
+	'Large', 
+	'Huge', 
+	'Gargantuan', 
+	'Colossal'
+);
+
+CREATE TYPE Alignment AS ENUM (
```

Comment:
Thought: do we want to have alignment set up like this or as two fields: good/evil and lawful/chaotic?

Reply:
The Pathfinder SRD has moments where it's easier to treat alignments as two separate axes, and moments where treating them as 9 elements of the same ilk is easier. It really depends on which problems we'd rather deal with - splitting one element into two, or remerging two disparate elements into one. I lean towards the former.